src/components/misc/side-panel.tsx

```diff
@@ -1,15 +1,36 @@
 import * as React from 'react';
+import * as Redux from "react-redux";
+import {Icon as IIcon, Store} from "../../redux/store-interfaces";
+import Icon from './icon';
 import './misc.less';
 
-class SidePanel extends React.Component {
+interface PropTypes { selectedIcon: IIcon };
+
+class SidePanel extends React.Component<PropTypes>{
 
     public render() {
+
+        const {selectedIcon} = this.props;
+        if(!selectedIcon){
+            return (
+                <div className="icon-side-panel">
+                    <h2>Ikke selektert</h2>
+                </div>
+            );
+        }
         return (
             <div className="icon-side-panel">
-                <h2>ToDo: Informasjon</h2>
+                <h2>{selectedIcon.title}</h2>
+                <Icon icon={selectedIcon} iconClass="selected-icon" iconContainerClass="selected-icon-container" iconShowDescription={false}/>
```

Comment:
Tror Icon componenten nå vil få et litt for stort ansvars område. Nå viser den både Icon-er og SelectedIconer. Jeg tror det kan være lurt å dele ansvaret opp i 2 komponenter. Så 1 har ansvaret for å vise SelectedIcon og 1 for å bare vise Iconet. Er også mulig å bruke Icon componenten i SelectedIcon componenten. For den skal jo tross alt vise iconet + litt mer